### ISSUE TITLE
[3.5.x-dev] GraphQL: Error Argument 2 passed to craft\\services\\Elements::_eagerLoadElementsInternal() must be of the type integer 

### ISSUE THREAD

**Issue by wsydney76** [craftcms/cms#6120]
### Description

Stoked to see the history button, but running into an error now:

```
{
  "errors": [
    {
      "debugMessage": "Argument 2 passed to craft\\services\\Elements::_eagerLoadElementsInternal() must be of the type integer, string given, called in E:\\craft\\multi\\vendor\\craftcms\\cms\\src\\services\\Elements.php on line 1711",
      "message": "Internal server error",
      "category": "internal",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "entries"
      ]
    }
  ],
  "data": {
    "entries": null
  }
}
```

The query: 

```
{
  entries(section: "film") {
    title
    ... on film_film_Entry {
      cast {
        ... on cast_role_BlockType {
          roleName
          persons {
            title
            url
          }
        }
      }
    }
  }
}

```

### Steps to reproduce

1. Issue the query in GraphQL Explorer
2.

### Additional info

- Craft version: 3.5.x-dev (8678b515a5)
- PHP version:
- Database driver & version:
- Plugins & versions:

**Comment by andris-sevcenko** [comment]
Can you send over your database dump, composer.json file, and composer.lock file to support@craftcms.com and reference this issue? Going to be much easier for us to find the issue that way.

**Comment by wsydney76** [comment]
@andris-sevcenko Done.

Thanks.

**Comment by brandonkelly** [comment]
This is fixed now, via 93c3b49dc1f244f30c0cf7cfe608e4aa3559ab39.
